draft-ietf-ppm-dap.md

```diff
@@ -2279,20 +2244,22 @@ Next, the Helper checks that:
 If any of these checks fail, then the Helper MUST abort with error
 `invalidMessage`. Additionally, if any prep step appears out of order relative
 to the previous request, then the Helper MAY abort with error `invalidMessage`.
-(Note that a report may be missing, in which case the Helper should assume the
-Leader rejected it.)
-
-Next, the Helper checks if the continuation step indicated by the request is
-correct. (For the first `AggregationJobContinueReq` the value should be `1`; for
-the second the value should be `2`; and so on.) If the Leader is one step behind
-(e.g., the Leader has resent the previous HTTP request), then the Helper MAY
-attempt to recover by sending the same response as it did for the previous
-`AggregationJobContinueReq`, without performing any additional work on the
-aggregation job. In this case it SHOULD verify that the contents of the
-`AggregationJobContinueReq` are identical to the previous message (see
-{{aggregation-step-skew-recovery}}). Otherwise, if the step is incorrect or if
-the Helper does not wish to attempt recovery, the Helper MUST abort with error
-`stepMismatch`.
+A report may be missing, in which case the Helper should assume the Leader
+rejected it.
+
+Next, the Helper checks the continuation step indicated by the request. If the
+`step` value is one greater than the job's current step, then the Helper
+proceeds.
+
+If it is equal to the current step (e.g., the Leader has resent the previous
+request), then the Helper MAY attempt to recover by sending the same response as
+it did for the previous `AggregationJobContinueReq`, without performing any
+additional work on the aggregation job. In this case it SHOULD verify that the
+contents of the `AggregationJobContinueReq` are identical to the previous
+message (see {{aggregation-step-skew-recovery}}).
+
+If the Helper does not wish to attempt recovery, or if the step has some other
+value, the Helper MUST abort with error `stepMismatch`.
```

Comment:
clarity: I think this is more clear as more paragraphs and is more explicit about what it means for a step to be "incorrect".